### PR TITLE
Redid order of project posts, added Dristhi and Potential Regional Us…

### PR DESCRIPTION
**Pull request by Nasafato**
Redid order of project posts, added Dristhi and Potential Regional Use of Africa's Natural Gas projects

### CODE REVIEW

**Comment by vr2262** [comment]
@Nasafato these posts were out of order before?

**Comment by Nasafato** [comment]
Yeah, they wee different from the current SEL page.

**Comment by vr2262** [comment]
@Nasafato how did that happen? A problem with the conversion process?

**Comment by vr2262** [comment]
@Nasafato ping

File: projects/_posts/2015-12-27-dristhi.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+id: 3250
+title: Dristhi
+author: Roger Wong
```
Comment (vr2262) [inline projects/_posts/2015-12-27-dristhi.md]: Due to #74 this should be

```
author: rowo
```

File: projects/_posts/2015-12-30-potential-regional-use-natural-gas.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+id: 3864
+title: Potential for Regional Use of East Africa’s Natural Gas
+author: Candice Heberer
```
Comment (vr2262) [inline projects/_posts/2015-12-30-potential-regional-use-natural-gas.md]: Due to #74 this should be

```
author: caheberer-ei
```

**Pull request closed without merging**